app/audit/models/ChargeSummaryAudit.scala

```diff
@@ -135,6 +152,8 @@ case class ChargeSummaryAudit(mtdItUser: MtdItUser[_], docDateDetail: DocumentDe
     ("interestPeriod", interestPeriod) ++
     ("dueDate", dueDate) ++
     ("fullPaymentAmount", fullPaymentAmount) ++
+    Json.obj( "endTaxYear" -> dateService.getCurrentTaxYearEnd) ++
```

Comment:
are you sure you need to call this via dateService but not to extract this from data itself?